concent_api/utils/helpers.py

```diff
@@ -17,3 +22,27 @@ def is_base64(data: str) -> bool:
 
 def decode_key(key: str) -> bytes:
     return base64.b64decode(key.encode('ascii'), validate = True)
+
+
+def get_current_utc_timestamp() -> int:
+    """
+    Returns current timestamp as int.
+    Returned timestamp is guaranteed to be in UTC as Django server is set to be in UTC.
```

Comment:
1) You mean that they will be in UTC only if `USE_TZ` is set to `True`?

    This function should return the time in UTC no matter what `USE_TZ` is set to. Or, if that's hard to do, we should simply say that we don't support `USE_TZ == False` by adding `assert settings.USE_TZ` as a precondition.

2) You did not mention in the docstring whether it's OS-dependent or not.

Reply:
Ad 1) I have checked and without this setting it still returns utc, so I will update comment.
Ad 2) Added.